### PR TITLE
Restricted Symfony update to 6.4.6 due to issue in latest version 6.4.7

### LINKED ISSUE #361: Update symfony minimum version to 6.4
## Description
As the Symfony 6.3 version support has reached its end of support and a new symfony version 6.4 is released which will be a long-term support till Nov. 2027, we will update the minimum requirement to ^6.4.*

Original Issue - https://github.com/symfony/symfony/issues/54983 - Symfony Serializer adding a conflict as 6.4.7creates above mentioned issue

### PR DESCRIPTION
**Pull request by kedarkhaire**
Fixes #361

### CODE REVIEW

**Review by divya-intelli — COMMENTED** [review]
Please don't restrict the version.

File: composer.json [hunk 1]
```diff
@@ -28,10 +28,17 @@
...
-        "symfony/options-resolver": "^6.3",
-        "symfony/property-access": "^6.3",
-        "symfony/property-info": "^6.3",
-        "symfony/serializer": "^6.3.11"
+        "symfony/options-resolver": "6.4.*",
+        "symfony/property-access": "6.4.6",
+        "symfony/property-info": "6.4.*",
+        "symfony/string": "6.4.*",
+        "symfony/var-exporter": "6.4.6",
+        "symfony/stopwatch":  "6.4.*",
```
Comment (divya-intelli) [inline composer.json]: Do we need this?

**Comment by mxr576** [comment]
If there is an issue with symfony/[package] 6.4.7, just add a `conflict` rule to package(s) and that version, nothing more.

**Comment by mxr576** [comment]
Also referencing the related issue - at least here -from Symfony's issue queue would be beneficial.

File: composer.json (hunk 1 above)
Comment (kedarkhaire) [reply composer.json]: Removed

**Comment by mxr576** [comment]
> Also referencing the related issue - at least here -from Symfony's issue queue would be beneficial.

Could this be related to this?

https://github.com/symfony/symfony/issues/54983

**Comment by kedarkhaire** [comment]
Yes @mxr576 that can be the related to the mentioned issue as only `symfony/serializer` with version 6.4.7 is causing issue.

**Review by divya-intelli — APPROVED** [review]
LGTM

**Pull request merged**